### PR TITLE
docs: simplify wording

### PR DESCRIPTION
**Pull request by marionebl**
This removes twisted germanisms from the mission statement. 
Also works on #1.

### CODE REVIEW

File: mission-statement.md [hunk 1]
```diff
@@ -1,17 +1,20 @@
...
-we know that and will account for the risks. New paths will be hard and inconvenient –
-we accept these hardships to create the best projects possible, for our colleagues and customers.
+we know that and will account for the risks. New path are inconvenient –
+we accept the hardships to create the best projects, for our colleagues and customers.
 
-We take on the hard fights. Most of the time we go where the big and hard problems are, this includes
-projects with unwieldy legacy code bases, encrusted processes or gigantic scopes.
+We take on the hard fights.
+We tame unwieldy legacy code bases,
+force open encrusted processes and
```
Comment (KnisterPeter) [inline mission-statement.md:15]: What do you mean by "force open encrusted processes".
You want to say that we open up encrusted processes?

File: mission-statement.md [hunk 2]
```diff
@@ -1,17 +1,20 @@
...
+deconstruct gigantic scopes.
```
Comment (KnisterPeter) [inline mission-statement.md:16]: This is a bit over the top from my perspective.
Lets find a more realistic wording here.

**Review by KnisterPeter — CHANGES_REQUESTED** [review]

File: mission-statement.md (hunk 1 above)
Comment (marionebl) [reply mission-statement.md:15]: "Force open" as in "He forced open a lock". I intended this to sound violent - "open up" would be the tamer variant. Your suggestion works, too.

File: mission-statement.md (hunk 2 above)
Comment (marionebl) [reply mission-statement.md:16]: What exactly feels over the top? The 

> ... legacy code bases, encrusted processes, gigantic scopes

are present in the current version.

File: mission-statement.md (hunk 1 above)
Comment (KnisterPeter) [reply mission-statement.md:15]: If you intended to sound that violent, then :+1:

File: mission-statement.md (hunk 2 above)
Comment (KnisterPeter) [reply mission-statement.md:16]: I've meant 'gigantic scopes'.
You are right, I've read over that phrase.

**Review by KnisterPeter — APPROVED** [review]

**Comment by marionebl** [comment]
@kotzendekrabbe @dipe @helgri Please review, I'd like to go forward on this.

**Comment by marionebl** [comment]
Going forward with this.

**Pull request merged**